wagn/lib/wagn/generators/wagn/wagn_generator.rb

```diff
@@ -28,10 +28,10 @@ class WagnGenerator < Rails::Generators::AppBase
   
 ## should probably eventually use rails-like AppBuilder approach, but this is a first step.  
   def dev_setup
-    @gem_path = options['gem-path']
+    @gem_path = ENV['WAGN_DEV_GEM_PATH'] || options['gem-path']
```

Comment:
Instead of --gem-path, or as one of the ways to set it?  Or maybe what you get if you don't set it?